modules/ocf/files/puppet-trigger

```diff
@@ -83,6 +102,9 @@ def main(argv=None):
         return 1
 
     if args.environment:
+        if args.environment != 'production' and not args.reason:
+            log('You must specify a reason to switch to a non-production environment')
```

Comment:
Maybe reword this to point to the `--reason` flag? Something like `You must specify a reason with the --reason flag to switch off of the production environment`.